### ISSUE TITLE
Sign In & Register Tab - Password length mis match issue.

### ISSUE THREAD

**Issue by santoshmondal** [AnthonyNahas/ngx-auth-firebaseui#297]
### Bug Report or Feature Request (mark with an `x`)
```
- [x] bug report -> please search issues before submitting
- [ ] feature request
```

### OS and Version?

macOS Sierra

### Versions

Angular CLI: 8.0.1
Node: 10.15.3
OS: darwin x64
Angular: 8.0.1

### Repro steps

[https://ngx-auth-firebaseui.firebaseapp.com/examples/appearance](https://ngx-auth-firebaseui.firebaseapp.com/examples/appearance)

**Comment by AnthonyNahas** [comment]
can you post a snippet of your code ?

**Comment by santoshmondal** [comment]
[https://ngx-auth-firebaseui.firebaseapp.com/examples/appearance](https://ngx-auth-firebaseui.firebaseapp.com/examples/appearance)

The above link is from the Example demo link you shared. There you can find the max length of password in Sign In tab is (0/25) and in the Register tab is (0/30).

Following is the snippet of the code from my project : 
```
<ngx-auth-firebaseui 
            [signInCardTitleText]="'Sign In'"
            [resetPasswordTabText]="'Password Recovery'"
            [providers]="[]"
            [registrationEnabled]="true"
            [resetPasswordEnabled]="true"
            [guestEnabled]="false"
            [tabIndex]="0"
            (onSuccess)="processLogin($event)"
            (onError)="processError($event)"    
            appearance="outline">
</ngx-auth-firebaseui>
```

**Comment by santoshmondal** [comment]
I have gone through the project source code. Following is the snippet from Sign In Tab, the things are hard coded here - 

```
<mat-form-field [appearance]="appearance">
    <mat-label>{{passwordText}}</mat-label>
    <input matInput
            type="password"
            minlength="6"
            maxlength="25"
            formControlName="password"
            required>
    <mat-icon matSuffix [color]="color">lock</mat-icon>
    <mat-hint align="end" aria-live="polite">
        {{signInFormGroup.value.password.length}} / 25
    </mat-hint>
    <mat-error *ngIf="sigInPasswordFormControl.hasError('required')">
        {{passwordErrorRequiredText}}
    </mat-error>
    <mat-error *ngIf="sigInPasswordFormControl.hasError('minlength')">
        The password must be at least 6 characters long.
    </mat-error>
    <mat-error *ngIf="sigInPasswordFormControl.hasError('maxlength')">
        The password must not exceed 25 characters
    </mat-error>
</mat-form-field>
```

**Comment by AnthonyNahas** [comment]
you have right! This should be patched

**Comment by AnthonyNahas** [comment]
this has been fixed in [v3.3.0](https://www.npmjs.com/package/ngx-auth-firebaseui) (update by @jeandat 👍)
